packages/utils/src/route.js

```diff
@@ -136,7 +136,8 @@ export const createRoutes = function createRoutes ({
   srcDir,
   pagesDir = '',
   routeNameSplitter = '-',
-  supportedExtensions = ['vue', 'js']
+  supportedExtensions = ['vue', 'js'],
+  trailingSlashes = undefined
 }) {
   const routes = []
   files.forEach((file) => {
```

Comment:
(Not related to this PR) what do you think of refactoring `forEach` to ES6 `for of`? OR using `files.map` to directly assign the result of `routes` instead of pushing to the end.

Reply:
Plus one for using `map` instead :relaxed:

Reply:
Let's do this later and not blocking this PR :)